src/inverseRobot.js

```diff
@@ -3,11 +3,25 @@
 /*
  * @param {object} robot
  *
- * @return {object}
+ * @return {object | null}
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const fixed = {};
+  const valuesSet = new Set();
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (valuesSet.has(value)) {
+      return null;
+    }
+
+    fixed[value] = key;
```

Comment:
Try to simplify your code, we don't need the `valuesSet` and the `fixed` object. We can check if the `value` already exists as a key in the `fixed` object using `hasOwnProperty`.